ui/raidboss/data/06-ew/raid/p4n.ts

```diff
@@ -38,7 +37,7 @@ const triggerSet: TriggerSet<Data> = {
       id: 'P4N Well Pinax',
       type: 'StartsUsing',
       netRegex: NetRegexes.startsUsing({ id: '6A3E', source: 'Hesperos', capture: false }),
-      delaySeconds: 1,
+      delaySeconds: 4,
```

Comment:
4 is reasonable for a 9 second cast (as I try to do 5 second warning for knockback).  One thing you can do here is do `delaySeconds: (_data, matches) => parseFloat(matches.castTime) - 5`, to have it be based on the cast time directly.

Reply:
Hm that is fancier but is the cast time likely to change? I'll keep it in mind for the future though, makes things easier.

Or do you want this changed for this trigger?

Reply:
No, I think it's fine to just say 4 if you'd like.